job/handlers/executeScript.js

```diff
@@ -57,7 +57,13 @@ function _executeTask(bag, next) {
   var who = bag.who + '|' + _executeTask.name;
   logger.debug(who, 'Inside');
 
-  var exec = spawn('/bin/bash', ['-c', bag.scriptPath + ' 2>&1'], bag.options);
+  var exec;
+  if (global.config.shippableNodeOperatingSystem === 'WindowsServer_2016') {
+    exec = spawn('powershell', [bag.scriptPath], bag.options);
```

Comment:
We redirect the stderr stream to stdout below as we want to handler everything in the stdout event. Is this not required here?